packages/ethereum/ethereum-types/src/consensus/merkle.rs

```diff
@@ -5,11 +5,11 @@
 
 // https://github.com/ethereum/consensus-specs/blob/dev/ssz/merkle-proofs.md
 /// `get_generalized_index(BeaconState, "finalized_checkpoint", "root")`
-pub const FINALIZED_ROOT_INDEX: u64 = 105;
+pub const FINALIZED_ROOT_GINDEX_ELECTRA: u64 = 169;
 /// `get_generalized_index(BeaconState, "current_sync_committee")`
-pub const CURRENT_SYNC_COMMITTEE_INDEX: u64 = 54;
+pub const CURRENT_SYNC_COMMITTEE_GINDEX_ELECTRA: u64 = 86;
 /// `get_generalized_index(BeaconState, "next_sync_committee")`
-pub const NEXT_SYNC_COMMITTEE_INDEX: u64 = 55;
+pub const NEXT_SYNC_COMMITTEE_GINDEX_ELECTRA: u64 = 87;
```

Comment:
ditto for all these to be made private and be used through a function. It will also be easier to make these changes in future hardforks